File: BigRedCup2018/F/main.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    for (int i = 0; i < 9; i++) {
        cout << i << endl;
        cout << flush;
        string t;
        getline(cin, t);
        if (t == "great!" || t == "don't think so" || t == "don't touch me!" || t == "not bad" || t == "cool") {
            cout << "normal" << endl;
            break;
        } else if (t == "don't even" || t == "are you serious?" || t == "go die in a hole" || t == "worse" ||
                   t == "terrible" || t == "no way") {
            cout << "grumpy" << endl;
            break;
        }
    }
}

File: BigRedCup2018/F/main.py
normal = ['great', "don't think so", "don't touch me",
          "not bad", "cool"]
grumpy = ["don't even", "are you serious", "go die in a hole",
          "worse", "terrible", "noway"]
for i in range(10):
    print(i, flush=True)
    v = input().lower()
    done = False
    for w in normal:
        if w in v:
            print("normal")
            done = True
            break
    for w in grumpy:
        if w in v:
            print("grumpy")
            done = True
            break
    if done:
        break
